### PR TITLE
[cookbook] update landing page

### PR DESCRIPTION
**Pull request by adpthegreat**
Added a table of contents to the cookbook, with each table containing the list of the pages in each of its respective sections, and the client code that is included in each of those cookbook pages. (i.e. web3js, rust, etc). 

This pull request was created for https://app.gib.work/bounties/e7194501-26e0-4fca-8c10-b56908465011 in an attempt to solve a bounty #627 . Payment for the bounty is immediately sent to the contributor after merge.
 
Fixes: #627

### CODE REVIEW

**Comment by adpthegreat** [comment]
hey @nickfrosty kindly review , thank you

**Review by nickfrosty — CHANGES_REQUESTED** [review]
@adpthegreat at a glance this looks super great. 

there are several things that are contrary to the [CONTRIBUTING doc](https://github.com/solana-foundation/developer-content/blob/main/CONTRIBUTING.md#heading-styles), like the heading and use of links.

Please remove the heading and fix the links

**Comment by nickfrosty** [comment]
@adpthegreat the links are still incorrect

you have them all like this:
````md
[Getting Test SOL](/development/test-sol.md)
````

they should all be like this (per [CONTRIBUTING#links](https://github.com/solana-foundation/developer-content/blob/main/CONTRIBUTING.md#links)):
````md
[Getting Test SOL](/content/cookbook/development/test-sol.md)
````

**Comment by adpthegreat** [comment]
alright will fix that

**Comment by adpthegreat** [comment]
@nickfrosty fixed that , any other thing to fix? thank you

File: content/cookbook/index.md [hunk 1]
```diff
@@ -7,7 +7,104 @@ description:
...
+| [How to Restore a Keypair](/content/cookbook/wallets/restore-keypair.md)                       | web3.js            | Recover existing keypairs           |
+| [How to Verify a Keypair](/content/cookbook/wallets/verify-keypair.md)                         | web3.js            | Validate keypair authenticity       |
+| [How to Validate a Public Key](/content/cookbook/wallets/check-publickey.md)                   | web3.js            | Check public key validity           |
+| [How to Generate Mnemonics for Keypairs](/content/cookbook/wallets/generate-mnemonic.md)       | bip39              | Create seed phrases                 |
+| [How to Restore a Keypair from a Mnemonic](/content/cookbook/wallets/restore-from-mnemonic.md) | web3.js, bip39     | Recover keypairs using seed phrases |
+| [How to Generate a Vanity Address](/content/cookbook/wallets/generate-vanity-address.md)       | Solana CLI         | Create custom addresses             |
+| [How to Sign and Verify a Message](/content/cookbook/wallets/sign-message.md)                  | web3.js, tweetNaCl | Message signing and verification    |
```
Comment (nickfrosty) [inline content/cookbook/index.md:42]:
```suggestion
| [How to Sign and Verify a Message](/content/cookbook/wallets/sign-message.md)                  | web3.js | Message signing and verification    |
```

File: content/cookbook/index.md [hunk 2]
```diff
@@ -7,7 +7,104 @@ description:
...
+| [How to Send SOL](/content/cookbook/transactions/send-sol.md)                                    | web3.js            | Transfer SOL between accounts  |
+| [How to Send Tokens](/content/cookbook/transactions/send-tokens.md)                              | web3.js            | Transfer SPL tokens            |
+| [How to Calculate Transaction Cost](/content/cookbook/transactions/calculate-cost.md)            | web3.js            | Estimate transaction fees      |
+| [How to Add a Memo to a Transaction](/content/cookbook/transactions/add-memo.md)                 | web3.js            | Include memos in transactions  |
+| [How to Add Priority Fees to a Transaction](/content/cookbook/transactions/add-priority-fees.md) | web3.js            | Set transaction priorities     |
+| [How to Optimize Compute Requested](/content/cookbook/transactions/optimize-compute.md)          | web3.js            | Improve transaction efficiency |
+| [Offline Transactions](/content/cookbook/transactions/offline-transactions.md)                   | web3.js, tweetNaCl | Handle offline operations      |
```
Comment (nickfrosty) [inline content/cookbook/index.md:57]:
```suggestion
| [Offline Transactions](/content/cookbook/transactions/offline-transactions.md)                   | web3.js | Handle offline operations      |
```

File: content/cookbook/index.md [hunk 3]
```diff
@@ -7,7 +7,104 @@ description:
...
+
+## Account Management
+
+Learn how to manage Solana accounts effectively.
+
+| Guide                                                                                     | Client        | Description                |
+| ----------------------------------------------------------------------------------------- | ------------- | -------------------------- |
+| [How to Create an Account](/content/cookbook/accounts/create-account.md)                  | web3.js       | Create new Solana accounts |
+| [How to Calculate Account Creation Cost](/content/cookbook/accounts/calculate-account.md) | web3.js       | Estimate account costs     |
```
Comment (nickfrosty) [inline content/cookbook/index.md:66]:
```suggestion
| [How to Calculate Account Creation Cost](/content/cookbook/accounts/calculate-rent.md) | web3.js       | Estimate account costs     |
```

File: content/cookbook/index.md [hunk 4]
```diff
@@ -7,7 +7,104 @@ description:
...
+| [How to Create a PDA's Account](/content/cookbook/accounts/create-pda.md)                 | web3.js, Rust | Work with PDAs             |
```
Comment (nickfrosty) [inline content/cookbook/index.md:67]:
```suggestion
| [How to Create a PDA's Account](/content/cookbook/accounts/create-pda-account.md)         | web3.js, Rust | Work with PDAs             |
```

File: content/cookbook/index.md [hunk 5]
```diff
@@ -7,7 +7,104 @@ description:
...
+| [Reading Multiple Instructions](/content/cookbook/programs/read-multiple-instructions.md)    | Rust   | Handle multiple instructions |
+| [How to Verify Accounts in a Solana Program](/content/cookbook/programs/verify-accounts.md)  | Rust   | Account verification         |
+
+## Token Operations
+
+Comprehensive guides for working with tokens on Solana.
+
+| Guide                                                                                                     | Client             | Description                    |
+| --------------------------------------------------------------------------------------------------------- | ------------------ | ------------------------------ |
+| [How to Create a Token](/content/cookbook/tokens/create-token.md)                                         | web3.js, spl-token | Create new tokens              |
```
Comment (nickfrosty) [inline content/cookbook/index.md:93]:
```suggestion
| [How to Create a Token](/content/cookbook/tokens/create-mint-account.md)                                  | web3.js, spl-token | Create new tokens              |
```

File: content/cookbook/index.md [hunk 6]
```diff
@@ -7,7 +7,104 @@ description:
...
+| [How to Create a Program Derived Address](/content/cookbook/programs/create-pda.md)          | Rust   | Generate PDAs                |
+| [How to Read Accounts in a Program](/content/cookbook/programs/read-accounts.md)             | Rust   | Account data access          |
+| [Reading Multiple Instructions](/content/cookbook/programs/read-multiple-instructions.md)    | Rust   | Handle multiple instructions |
+| [How to Verify Accounts in a Solana Program](/content/cookbook/programs/verify-accounts.md)  | Rust   | Account verification         |
+
+## Token Operations
+
+Comprehensive guides for working with tokens on Solana.
+
+| Guide                                                                                                     | Client             | Description                    |
```
Comment (nickfrosty) [inline content/cookbook/index.md:91]: please also remove all the `spl-token` and `metaplex` labels in the client column of all of these

**Review by nickfrosty — CHANGES_REQUESTED** [review]
Thanks for adding this, there are some broken links and mislabled "Clients".

**Comment by adpthegreat** [comment]
@nickfrosty done that, I've fixed the broken links and mislabeled clients

**Review by nickfrosty — APPROVED** [review]

**Pull request merged**